cucumber/policy/features/step_definitions/error_steps.rb

```diff
@@ -13,11 +13,19 @@
 end
 
 Then(/^there is an error$/) do
-  json = json_result
-  expect(json).to have_key('error')
-  @error = json['error']
+  expect(json_result).to have_key('error')
+  @error = json_result['error']
 end
 
 Then(/^there is no error$/) do
   expect(json_result).not_to have_key('error')
 end
+
```

Comment:
Since the difference between this step and the two steps above is going to confuse people, we should add a comment stating the reason we did it, along with the plan to refactor the other references to the non-apostrophe version, and then ultimately delete this.